<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieFileRename</class>
 <widget class="QWidget" name="MovieFileRename">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1478</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>1441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>811</x>
      <y>10</y>
      <width>631</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="testButton">
       <property name="text">
        <string>Get Folder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="run_button">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>Next File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renameButton">
       <property name="text">
        <string>Re Name Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="QAButton">
       <property name="text">
        <string>QA Check</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>72</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_47">
         <property name="text">
          <string>Enter a Movie Name - Click Next File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="checkBox_11">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ignore pre checked library files</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>135</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="sliderPosition">
        <number>80</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Select file comparison %</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="levRatioEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="masterFileNameTxt">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Preferred File Name without extension</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1441</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="11" column="3">
     <widget class="QLabel" name="namelabel_10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="namelabel_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="namelabel_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QLabel" name="namelabel_8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="namelabel_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="namelabel_7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="4" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="rencheckBox_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="namelabel_6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="parentButton_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="4" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="rencheckBox_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="pathlabel_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="pathlabel_1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignBottom">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select Parent</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="parentButton_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QLabel" name="namelabel_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="recordIDlabel_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="4" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="rencheckBox_6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="recordIDlabel_1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="namelabel_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="pathlabel_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="4" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="rencheckBox_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="recordIDlabel_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="pathlabel_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="parentButton_1">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="rencheckBox_1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="4" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="rencheckBox_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="4" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="rencheckBox_7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="4" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="rencheckBox_10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLabel" name="pathlabel_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="pathlabel_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="4" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="rencheckBox_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QLabel" name="pathlabel_10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLabel" name="chlabel_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLabel" name="chlabel_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="4" alignment="Qt::AlignHCenter">
     <widget class="QCheckBox" name="rencheckBox_8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="chlabel_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLabel" name="chlabel_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QLabel" name="chlabel_7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="pathlabel_6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QLabel" name="chlabel_6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="0" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="parentButton_8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="0" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="parentButton_6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLabel" name="chlabel_1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="recordIDlabel_10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="pathlabel_7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="namelabel_1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="0" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="parentButton_10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QLabel" name="pathlabel_8">
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="parentButton_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="0" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="parentButton_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="0" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="parentButton_7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QLabel" name="chlabel_8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="recordIDlabel_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="recordIDlabel_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="11" column="5">
     <widget class="QLabel" name="chlabel_10">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="recordIDlabel_8">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="recordIDlabel_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="recordIDlabel_6">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="5">
     <widget class="QLabel" name="chlabel_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0" alignment="Qt::AlignHCenter">
     <widget class="QRadioButton" name="parentButton_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3" alignment="Qt::AlignBottom">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>File Name</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="recordIDlabel_7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1" alignment="Qt::AlignBottom">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Library ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" alignment="Qt::AlignBottom">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>File Path</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" alignment="Qt::AlignBottom">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select File for Action</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" alignment="Qt::AlignBottom">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Action Confirmation</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>48</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
